### DISCUSSION TITLE
Browser Mod crea incessantemente nuove entità

### DISCUSSION THREAD

**Discussion by Topolynx** [indomus/forum#397] [HUB: Home Assistant]
Ciao a tutti!
Tempo fa decisi di integrare la mia smart tv LG in Home Assistant seguendo [la guida di indomus](https://indomus.it/guide/integrare-e-definire-telecomandi-virtuali-tv-lg-webos-su-lovelace-ui-di-home-assistant/), tutto ok per quanto riguarda sia l'integrazione della tv sia per il telecomando nella UI.
E' successo però che l'integrazione browser_mod ha cominciato a sfornare entità a rotta di collo, così mi ritrovo attualmente con 107 entità, e ogni volta che le elimino pian piano ne riappaiono di nuove che finiscono per diventare non disponibili. Probabilmente ho inserito, o meglio NON ho inserito il dovuto codice nel configuration.yaml, dal momento che l'unica voce a riguardo è `browser_mod:` e basta.
Qualcuno di voi ha avuto problemi simili?
![Screenshot (9)](https://user-images.githubusercontent.com/19503090/120825787-06fe7500-c55a-11eb-9c01-16b86e794967.png)

**Comment by Scialla** [comment]
aggiungi la riga che segue, o meglio, configura così:
```
browser_mod:
  disable: all
  ```

**Reply by Topolynx** [reply]
Grazie!
